### PR TITLE
add pull request template for project

### PR DESCRIPTION
**Pull request by wolfstudy**
add pull request template for project, standardize everyone's pr process.

### CODE REVIEW

**Comment by wolfstudy** [comment]
@cch123 @rfyiamcool PTAL

File: .gitignore [hunk 1]
```diff
@@ -17,3 +15,15 @@ fuzz/pulsar-corpus-gen
 fuzz/*.zip
 fuzz/*-workdir/
 fuzz/initial-corpus/
+
+# IDE generated files
+.idea/
+
+# dep generated files
+vendor
```
Comment (cch123) [inline .gitignore]: ignore vendor?
what if the original repo was removed or renamed~

Comment (wolfstudy) [reply .gitignore]: Later, may consider replacing the `dep` with `go mod`

**Comment by rfyiamcool** [comment]
ok

**Pull request merged**